src/firecrest/status/health_check/health_checker_cluster.py

```diff
@@ -41,9 +44,13 @@ def __init__(self, cluster: HPCCluster, token_decoder: OIDCTokenAuth = None):
 
     async def check(self) -> None:
         try:
+            ctx = ssl.create_default_context(
+                cafile=os.environ.get("REQUESTS_CA_BUNDLE", certifi.where()),
+            )
             client = AsyncOAuth2Client(
                 self.cluster.service_account.client_id,
                 self.cluster.service_account.secret.get_secret_value(),
+                verify=ctx,
```

Comment:
Can we reduce the number of new imports by simplifying the change?
Would something like this work?
```
ca_bundle = os.environ.get("REQUESTS_CA_BUNDLE", None)
...
verify= ca_bundle if ca_bundle else True
```